### PR TITLE
[No ticket][Risk=no] Fixing notebook display names

### PR DESCRIPTION
**Pull request by UrsaStutsman**
Fixing notebook display names

### CODE REVIEW

File: ui/src/app/views/resource-card/component.ts [hunk 1]
```diff
@@ -190,9 +182,8 @@ export class ResourceCardComponent implements OnInit {
         break;
       }
       case ResourceType.NOTEBOOK: {
-        const nbUrl = '/workspaces/' + this.wsNamespace + '/' + this.wsId + '/notebooks/'
-          + encodeURIComponent(this.resourceCard.notebook.name);
-        window.open(nbUrl, '_blank');
+        this.route.navigate(['workspaces', this.wsNamespace, this.wsId, 'notebooks',
```
Comment (calbach) [inline ui/src/app/views/resource-card/component.ts]: I think this line can be reverted and included in your other PR (we still want to open a new tab for now)

File: ui/src/app/views/resource-card/component.ts [hunk 2]
```diff
@@ -154,28 +154,20 @@ export class ResourceCardComponent implements OnInit {
...
       case ResourceType.CONCEPT_SET: {
         this.conceptSetsService.deleteConceptSet(this.wsNamespace, this.wsId, $event.id)
-          .subscribe(() => {
-            this.onUpdate.emit();
-            this.deleteModal.close();
-          });
+          .subscribe(() => this.onUpdate.emit());
       }
     }
+    this.deleteModal.close();
```
Comment (calbach) [inline ui/src/app/views/resource-card/component.ts]: Was this change intentional? Looks like it may have reverted something that Bru did

File: ui/src/app/views/resource-card/component.ts (hunk 1 above)
Comment (UrsaStutsman) [reply ui/src/app/views/resource-card/component.ts]: I swear I included this change in my other PR. I'll double check now and add it back in if it got left out somehow.
